dls-controls/malcolmjs
# Feature/widget text input #26

## Description

Updated text input widget to have better state behaviour: now retains local state when "dirty", but tracks malcolm state when "clean"

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- connect to dev server (or simulator) on localhost:3000/gui/PANDA/layout/PANDA:PULSE1 (or some other block with a textInput widget) and explore behaviour 

## Agile board tracking

connect to #26
